dapps/onboarding.vue/src/components/sign-up/sign-up.ts

```diff
@@ -223,13 +226,13 @@ export default class SignUp extends mixins(EvanComponent) {
 
     this.termsAccepted = new EvanFormControl('termsAccepted', false, this);
 
-    // if the user was inivted, show the welcome page
-    if (this.$route.query.inviteeAlias) {
-      this.steps.push({
-        title: '_onboarding.sign-up.welcome',
-        disabled: () => this.activeStep !== 3,
-      });
-    }
+    /* if the user was inivted, show the welcome page
```

Comment:
was the code commented intentionally? If yes remove it completely or add a comment why it needs to be kept in comment.

Reply:
Ahh i forgot to remove it, is dropped now.